binding/java/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.12-bin.zip
```

Comment:
we can't update without significant testing as it can change what versions of java the SDK supports